### DISCUSSION TITLE
Bug report: New precise code navigation executes on the GitHub UI

### DISCUSSION THREAD

**Discussion by jstockwin** [community/community#8664] [Pull Requests]
The new Precise code navigation mentioned in this [blog post](https://github.blog/2021-12-09-precise-code-navigation-python-code-navigation-pull-requests/) has some interesting behaviour. See the below screenshot: 

![Selection_420](https://user-images.githubusercontent.com/8227759/145592153-4b7a692f-9ae8-4528-82a2-80ebfb27f924.png)

It seems to be trying to perform code navigation on the word `Update` from the `Update comment` button during a code review. Note the shown code is the first line of the selected file.

It seems fairly easy to reproduce - go onto the `Files` tab of a PR, go to add a new comment, and then hover over any of the text in a button. After a short time, the highlighting for code navigation appears and clicking will show the popup. I guess it might only work for Python if that's all the new code navigation works for.

Awesome feature otherwise! Let me know if you need any more details.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
